schema.go

```diff
@@ -97,13 +97,13 @@ func attrSchema(t reflect.Type) (*Value, error) {
 	}
 }
 
-func sliceToBlockSchema(t reflect.Type, tag tag) (*Block, error) {
+func sliceToBlockSchema(t reflect.Type, tag tag, opt *MarshalOptions) (*Block, error) {
```

Comment:
MarshalOptions should be a private type.

Reply:
Done